classes/Tools.php

```diff
@@ -1167,6 +1167,7 @@ public static function clearXMLCache()
      */
     public static function displayError($errorMessage = null, $htmlentities = null, Context $context = null)
     {
+        header('HTTP/1.1 500 Internal Server Error', true, 500);
```

Comment:
hi @jocel1 ,

this may occur some desagrements,
 it seem nice, but i think that often  the error message is shorter than an old IE min error message lenghth,
so in many case user will see the default browser 500 error instead of the real error,
https://perishablepress.com/important-note-for-your-custom-error-pages/

sorry, i havent test this today whith a recent browser, and i does not know if my comment is relevant?


regards

Reply:
Well those error messages are useful mainly for developers anyway, and I hope they don't use old IE :D

Reply:
Agreed with @jocel1 . More if an error occurred we mustn't return a HTTP 200.

Reply:
agree also

Reply:
Misspelling, wanna said `must` :smile: